rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -230,9 +235,15 @@ def _create_networks(self, network_create_args):
         physnet = network_create_args.get("physical_network", None)
         lswitches = self._create_lswitches(network_create_args)
 
+        install_method = network_create_args.get("install_method", None)
+        if not install_method:
+            print "No install_method, use default"
+            install_method = "sandbox"
```

Comment:
Yes, get() will return "sandbox" if "lswitch_create_args" has no "install_method" in it.